src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -356,6 +358,19 @@ private static Job CreateJobForGivenRuntime(Job baseJob, string runtimeId, Comma
                     builder.TargetFrameworkMoniker(runtime.MsBuildMoniker);
 
                     return baseJob.WithRuntime(runtime).WithToolchain(builder.ToToolchain());
+                case RuntimeMoniker.Wasm:
+                        IToolchain toolChain = new WasmToolChain("Wasm",
+                                                                 "net5.0",
+                                                                 options.CliPath.FullName,
+                                                                 null,
```

Comment:
using the provided packages restore path is important from CI perspective
```suggestion
                                                                 options.RestorePath?.FullName,
```